### ISSUE TITLE
Conda frontend value is not propagated to subworkflows

### ISSUE THREAD

**Issue by timtroendle** [snakemake/snakemake#939]
**Snakemake version**
6.0.5

**Describe the bug**
I choose `mamba` as conda frontend, however `conda` is used in the subworkflows. 

**Logs**
I added a log statements just behind the workflow creation in `__init__.py` to verify this. This is the log output:

```bash
Frontend: mamba
Building DAG of jobs...
Executing subworkflow mysubworkflow.
Frontend: conda
Building DAG of jobs...
```

**Comment by fgvieira** [comment]
I can confirm this bug is still present in version `6.4.1`, although the inverse (since the default frontend changed from `conda` to `mamba`). If I choose `conda` as conda frontend, `mamba` is used in the subworkflows.

**Comment by johanneskoester** [comment]
Thanks for reporting. Should be fixed in b1b951f5.
